### DISCUSSION TITLE
Allow Wlan to belong to multiple Wlan Groups

### DISCUSSION THREAD

**Discussion by BlueSkyGreenGrass** [netbox-community/netbox#8849] [Ideas]
My organization has multiple buildings spanning multiple cities that have various amounts of Wlans and Wlan groups.

My current problem is that a wlan can only belong to one specific wlan group. Where as in my organization we have multiple departments that sometimes are present with other departments - sometimes not. So we need to have those wlans belong to multiple different groups - depending on what the building calls for.

Currently we are just listing the APs in Netbox and not using the group function at all, just assigning the Wlans to the AP individually. But we would love the feature to have one Wlan belong to multiple groups and just link in the group.

Thank you for your time.

**Comment by nomad-cyanide** [comment]
But would that not in fact be seperate vlans for each site, even though they are matching in name, id, etc etc.? I think it is a different funtionality you are looking for.

We have a lot of vlans here, across multiple sites and vlangroups, even though the vlans have the same name, id and such, it is actually seperate vlans stretched across several L2 broadcast domains, bound together by the same VNI in a custom field (we are running IPfabric) . its a pain to maintain, but this is where automation comes in.
A vlan-group equals a L2 broadcast domain, as I understand it.

**Reply by candlerb** [reply]
> A vlan-group equals a L2 broadcast domain, as I understand it.

I'd disagree with that.  From my understanding, a vlan is a L2 broadcast domain; if it is bridged across multiple sites, it's the same vlan.  For that reason, a prefix can only be assigned to a single VLAN (although a VLAN can have multiple prefixes).

A vlan-group is an administrative collection of vlans.

**Comment by mtinberg** [comment]
In my case if I do have two dissimilar VLANs that are bridged together (through VLAN translation in many cases) then I have a custom field on VLAN with the ID number of the peer VLAN record and custom links to make it clickable to switch between records if this field is populated.  The Prefix(s) are then associated with the routed VLAN (SVI) record

eg.

{% if obj.custom_field_data.vlan_translate %}
Translated VLAN
{% endif %}

/ipam/vlans/{{ obj.custom_field_data.vlan_translate }}



—

Mark Tinberg ***@***.***>

Division of Information Technology-Network Services

University of Wisconsin-Madison
